beetsplug/spotify.py

```diff
@@ -64,6 +66,7 @@ def __init__(self):
                 'client_id': '4e414367a1d14c75a5c5129a627fcab8',
                 'client_secret': 'f82bdc09b2254f1a8286815d02fd46dc',
                 'tokenfile': 'spotify_token.json',
+                'force': False,
```

Comment:
Seems OK to make this a CLI-only flag for now, and omit the configuration option. It will be a little less confusing that way because the `force` option *only* applies to the new command, not the remaining functionality of the plugin.